src/app/modules/shared_models/token_model.nim

```diff
@@ -172,7 +176,7 @@ QtObject:
       of "change24hour": result = $item.getChange24hour()
       of "currencyPrice": result = $item.getCurrencyPrice()
       of "decimals": result = $item.getDecimals()
-      of "pegSymbol": result = $item.getPegSymbol()
```

Comment:
`pegSymbol` no longer needed? (still present elsewhere in the model)

Reply:
actually I am not sure what this stands for, I dont see a usage on UI side though, should I remove it?

Reply:
it tells you if a crypto is pegged to some fiat currency (so, for `DAI`, the `pegSymbol` will be `USD`). I used this info when we calculated currency formatting on the client side, but that's all done in `status-go` now so this doesn't get used here anymore. I don't mind if it gets removed or if it gets left in peace :D

Reply:
I'll remove it then